### PR TITLE
IDE-3580 create maven module project by using blade cli

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

**Comment by yanan** [comment]
Test Failed on Jenkins http://ide-build-1:9090/ on this commit 9bc02196b53441eaa8e8982642b17192729dcd81 on build 50

File: maven/tests/com.liferay.ide.maven.core.tests/src/com/liferay/ide/maven/core/tests/MavenModuleProjectTests.java [hunk 1]
```diff
@@ -589,49 +589,125 @@ public void testProjectTemplateFormField() throws Exception
     }
 
     @Test
-    public void testProjectTemplateFragment() throws Exception
```
Comment (gamerson) [inline maven/tests/com.liferay.ide.maven.core.tests/src/com/liferay/ide/maven/core/tests/MavenModuleProjectTests.java]: You removed the testProjectTemplateFragment, can you add it back?

File: maven/tests/com.liferay.ide.maven.core.tests/src/com/liferay/ide/maven/core/tests/MavenModuleProjectTests.java [hunk 2]
```diff
@@ -589,49 +589,125 @@ public void testProjectTemplateFormField() throws Exception
...
+    }
+
+    @Test
+    public void testProjectTemplateNpmVuejsPortlet() throws Exception
+    {
+        NewLiferayModuleProjectOp op = NewLiferayModuleProjectOp.TYPE.instantiate();
+
+        op.setProjectName( "npm-vuejs-portlet-test" );
+        op.setProjectProvider( "maven-module" );
+        op.setProjectTemplateName( "npm-vuejs-portlet" );
```
Comment (gamerson) [inline maven/tests/com.liferay.ide.maven.core.tests/src/com/liferay/ide/maven/core/tests/MavenModuleProjectTests.java]: Do we have unit tests for all project types?

File: maven/tests/com.liferay.ide.maven.core.tests/src/com/liferay/ide/maven/core/tests/MavenModuleProjectTests.java (hunk 1 above)
Comment (JoyeLuo) [reply maven/tests/com.liferay.ide.maven.core.tests/src/com/liferay/ide/maven/core/tests/MavenModuleProjectTests.java]: Ok, got it. ProjectTemplateFragment Junit test is written in  MavenModuleFragmentProjectTests. And I will also add this test back in MavenModuleProjectTests.

File: maven/tests/com.liferay.ide.maven.core.tests/src/com/liferay/ide/maven/core/tests/MavenModuleProjectTests.java (hunk 2 above)
Comment (JoyeLuo) [reply maven/tests/com.liferay.ide.maven.core.tests/src/com/liferay/ide/maven/core/tests/MavenModuleProjectTests.java]: For maven module, we have unit tests for all project types now. 
For gradle module, we don't have it. I will add it.

**Pull request closed without merging**